.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
don't include this file in pr's its vs-code specific and doesn't require to be in the codebase.